node-api/src/events/raw_event_details.rs

```diff
@@ -326,3 +326,535 @@ impl<Hash: Encode + Decode> RawEventDetails<Hash> {
 		Ok(decoded)
 	}
 }
+
```

Comment:
hmm, not sure if we need all these tests. What I meant was just the high level one you added below ...

Reply:
I know. But since I was at it already and needed to create new EventDetails either way, I copied & adapted the other unit tests. Is that not acceptable?

Reply:
No, it's ok now